sklearn/manifold/t_sne.py

```diff
@@ -507,8 +507,7 @@ class TSNE(BaseEstimator):
         The perplexity is related to the number of nearest neighbors that
         is used in other manifold learning algorithms. Larger datasets
         usually require a larger perplexity. Consider selecting a value
-        between 5 and 50. While the choice is not extremely critical, results
-        have been shown to change considerably across perplexity values.
+        between 5 and 50. Different values can result in significanlty different results.
```

Comment:
@NicolasHug the one I previously proposed made it seem like it was suggesting that changing the value from the default (30) would impact the results, hence this wording.

Reply:
The wording is fine to me, but the checks are failing since the line is too long. Please wrap around 79 characters